modules/maven/default/module.yaml

```diff
@@ -31,11 +31,21 @@ envs:
 - name: MAVEN_CLEAR_REPO
   description: If set then the Maven repository is removed after the artifact is built. This is useful for keeping the created application image small, but prevents *incremental* builds. Will be overridden by **S2I_ENABLE_INCREMENTAL_BUILDS**.  Defaults to *false*.
 - name: MAVEN_MIRROR_URL
-  description: The base URL of a mirror used for retrieving artifacts.
+  description: >
+    The base URL of a mirror used for retrieving artifacts.
+    See `MAVEN_MIRRORS` for specifying multiple mirrors.
```

Comment:
This is already part of #403. Wouldn't this conflict?

Reply:
Yes, whoops. Resolved now, thanks.